vrf-listener/benchmark/stress/stress_tester.py

```diff
@@ -68,7 +68,7 @@ async def run_stresstest(self) -> None:
         print("\n🧩 Creating user clients...")
         users = await self.config.get_users_client(
             network=self.network,
-            rpc_url=self.rpc_url,
+            rpc_url="https://api.cartridge.gg/x/starknet/sepolia",
```

Comment:
revert this change ? rpc should be configurable no?

Reply:
done